### PR TITLE
[PROCEDURES] Officially allow adding new datatypes to the latest tagged release

### PR DESCRIPTION
**Pull request by nsoranzo**
or to a frozen (but not yet tagged) release.

Also some language editing I had in my git stash.

### CODE REVIEW

File: doc/source/project/organization.rst [hunk 1]
```diff
@@ -80,18 +80,18 @@ The above list of people is the canonical source used to determine
...
-rules for pull requests that modify governance procedures outlined below - with
-one exception - a *committer* may not vote
+rules for pull requests that modify governance procedures outlined below, with
+one exception: a *committer* may not vote
 against their own removal from the group (for obvious reasons).
 
-Given the responsibilities and power invested in this group - it is important
-that individuals not actively working on Galaxy in some fashion are removed from
+Given the responsibilities and power invested in this group, it is important
+that individuals not actively working on Galaxy any more are removed from
```
Comment (jdavcs) [inline doc/source/project/organization.rst]:
```suggestion
that individuals not actively working on Galaxy anymore are removed from
```

File: doc/source/project/organization.rst [hunk 2]
```diff
@@ -80,18 +80,18 @@ The above list of people is the canonical source used to determine
...
 the group. If individuals in this group intend to change jobs or reallocate
 volunteer activities and will no longer be active in the Galaxy community,
 they should withdraw from membership of this group. Periodically, active
 members may review this group and request inactive members are removed - this
```
Comment (jdavcs) [inline doc/source/project/organization.rst]:
```suggestion
members may review this group and request that inactive members are removed - this
```

**Comment by nsoranzo** [comment]
I'll leave this open for more comments for a bit, then close and re-open as per our Procedures.

**Comment by ic4f** [comment]
Given that this is a [procedures] PR, both of my edits are very minor and can be ignored for the sake of getting this merged fast. So, unless there are more edits/suggestions, please ignore.

**Review by jdavcs — APPROVED** [review]

**Comment by nsoranzo** [comment]
> Given that this is a [procedures] PR, both of my edits are very minor and can be ignored for the sake of getting this merged fast. So, unless there are more edits/suggestions, please ignore.

I've accepted already your suggestions, so I have to close&re-open. No worries, there's no hurry.

**Review by bernt-matthias — APPROVED** [review]

**Review by bgruening — APPROVED** [review]

**Pull request closed without merging**